.golangci.yml

```diff
@@ -6,7 +6,6 @@ linters:
   enable-all: false
   disable-all: true
   enable:
-    - vet
```

Comment:
Why is this being disabled?

Reply:
my bad, vet was throwing ` Error: loopclosure: loop variable tt captured by func literal (govet)` for some test files.
Should I add `//nolint:govet` to those files for now and look at this in a separate issue?

Reply:
Can you try to see if it is a genuine issue? If not, we can add an exclusion in the golangci-lint configuration.

Reply:
that warning might be coming for a good reason. if your tests are running parallely, re-using loop variable can cause issue.

Reply:
Fixed this, thanks!